src/librustc/ty/constness.rs

```diff
@@ -55,7 +55,7 @@ impl<'a, 'tcx> TyCtxt<'a, 'tcx, 'tcx> {
             }
             // in order for a libstd function to be considered min_const_fn
             // it needs to be stable and have no `rustc_const_unstable` attribute
-            match self.lookup_stability(def_id) {
+            self.is_const_fn_raw(def_id) && match self.lookup_stability(def_id) {
```

Comment:
What is this change about (and same some lines down)?

Reply:
All calls to `is_min_const_fn` need to guarantee that the `DefId` is a a function. This way it dosn't need to, because the function checks it itself.